components/addTask.vue

```diff
@@ -47,14 +49,51 @@ import { defineComponent } from '@nuxtjs/composition-api'
 
 export default defineComponent({
   emits: ['newTask'],
+  data() {
+    return {
+      title: ''
+    }
+  },
   methods: {
-    addTask() {
+    async addTask() {
       /**
        * @todo Complete this function.
        * @todo 1. Send the request to add the task to the backend server.
        * @todo 2. Add the task in the dom.
        * @hint use emit to make a event that parent can observe
        */
+      let taskTitle = this.title.trim();
+      this.title = '';
+      let token = this.$store.getters.token;
+      //console.log(this.title)
+      if(taskTitle != '') {
+        const data = 
+        await this.$axios({
```

Comment:
error should be handled with try-catch if you are using await

Reply:
did it using try catch 👍🏻